user_guide/intro_nav_history_fav_viewed.rst

```diff
@@ -0,0 +1,64 @@
+.. _user-guide-getting-started-history:
+
+History, Favorites and Most Viewed
+==================================
+
+The "hamburger" icon next to the user menu contains a history of your page views, the list of your favorite pages, and 
+the list of the most viewed pages in OroCRM.
+
+All the page names available in the lists are clickable links to the relevant pages. 
+
+|
+
+.. image:: ./img/navigation/panel/hamburger.png  
+
+|
+
+History
+-------
+
+This section displays links to the latest 20 pages you’ve visited in the reverse chronological order. The list is 
+particularly useful to open a page that you have recently used without searching for it. 
+
+
+Favorites
+---------
+
+The section displays links to all the pages that you have marked as "favorite".
```

Comment:
Comment: A fairly minor point (and I may have made this earlier), but if you want to accord with American English, punctuation should appear within quotes, such as: "favorites."